src/openrct2/world/MapAnimation.cpp

```diff
@@ -199,8 +199,6 @@ static bool map_animation_invalidate_small_scenery(const CoordsXYZ& loc)
                         continue;
                     if (peep->z != loc.z)
                         continue;
-                    if (peep->Action < PeepActionType::None1)
-                        continue;
```

Comment:
Should this one be removed? It feels a bit disconnected from the rest of the PR.

Reply:
A few lines above it already checks for a specific action

Reply:
I don't think that's exactly the same it checks for a specific state but not a specific action.

Reply:
Yeah removing this is actually wrong, got confused over state.